camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -229,8 +231,27 @@ def handle_monocular(self, msg):
                                         checkerboard_flags=self.checkerboard_flags,
                                         max_chessboard_speed = self._max_chessboard_speed)
 
-        # This should just call the MonoCalibrator
+        # This should just call the MonoCalibrator and compute the mask for the detected corners if any
+        prec_progress = self._progress
         drawable = self.c.handle_msg(msg)
+        img = self.c.mkgray(msg)
```

Comment:
@brinij I was thinking on moving all the mask creation into the MonoCalibrator and add the mask as an attribute which could be retrieved, but before that I'd like to get some feedback

Reply:
Yes please do that, and maybe have an arg of a method the radius of removing the red mask at detected corners.

Reply:
addressed